scripts/modules/events/enhanced-integration-manager.js

```
@@ -0,0 +1,816 @@
+
+			case this.processingModes.BUS:
+				return this._emitToBus(eventType, payload, options);
+
+			case this.processingModes.HYBRID:
+				// Emit to both direct and bus systems
+				const results = await Promise.allSettled([
+					this._emitDirect(eventType, payload, options.directOptions),
+					this.eventBus
+						? this._emitToBus(eventType, payload, options)
+						: Promise.resolve(null)
+				]);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)